### PR TITLE
Improve callback command with an example

### PR DESCRIPTION
**Pull request by jadeallenx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,12 +113,39 @@ riak_cli:register_command(Cmd, KeySpecs, FlagSpecs, Fun).
...
 
 #### Command callback implementation
-The function which is registered as the callback for this command gets two arguments.
-One is a proplist of key/value pairs (if any, appropriately typecast as specified), and the 
-other is a proplist of flags (if any, also appropriately typecast).  The flags
-proplist contains the given "longname" converted to an atom as the proplist key.
+The function which is registered as the callback for this command gets two
+arguments.  One is a proplist of key/value pairs (if any, appropriately
+typecast as specified), and the other is a proplist of flags (if any, also
+appropriately typecast).  If the user provided the "shortname" of a flag, it is
```
Comment (andrewjstone) [inline README.md]: Hmmm, is this really true? If so it's a bug in riak_cli. I always intended to return the actual name of the key which is the atom (which should also match longname). I will look into it tonight as I finish up riak-admin describe.

Comment (andrewjstone) [inline README.md]: @mrallen1 I just checked this out, and I don't believe it to be true. The type of `flags` in `riak_cli_command:run/1` is defined as a proplist which is defined as only taking an `atom()` as the first term in each tuple here: https://github.com/basho/riak_cli/blob/develop/src/riak_cli_command.erl#L13
Dialyzer passes successfully. 

Also, the code that does the check and conversion is here: https://github.com/basho/riak_cli/blob/develop/src/riak_cli_parser.erl#L166

There are also tests for this: https://github.com/basho/riak_cli/blob/develop/src/riak_cli_parser.erl#L286

Comment (jadeallenx) [inline README.md]: OK, I based this on the comments in the test suite here:

https://github.com/basho/riak_cli/blob/develop/src/riak_cli_parser.erl#L275-L284

So I will update the PR with the right info. Thanks for validating the behavior.

Comment (jadeallenx) [inline README.md]: For the record:

``` erlang
(dev2@127.0.0.1)1> riak_cli:run(["riak-admin","cluster","members", "-a"]).
got flags: [{all,undefined}]
'dev1@127.0.0.1'
'dev2@127.0.0.1'
'dev3@127.0.0.1'
'dev4@127.0.0.1'
'dev5@127.0.0.1'
ok
```

You (and dialyzer) are correct :)

**Comment by jadeallenx** [comment]
Closing this in light of #21

**Pull request closed without merging**